helix_fhir_client_sdk/utilities/retryable_aiohttp_client.py

```diff
@@ -90,12 +91,17 @@ async def fetch(
         while retry_attempts < self.retries:
             retry_attempts += 1
             try:
-                if self.chunked:
-                    kwargs["chunked"] = self.chunked
-                if self.compress:
-                    kwargs["compress"] = self.compress
                 if headers:
                     kwargs["headers"] = headers
+                # if there is no data then remove from kwargs so as not to confuse aiohttp
+                if "data" in kwargs and kwargs["data"] is None:
+                    del kwargs["data"]
```

Comment:
aiohttp seem to be already checking for data None. Can you please elaborate what case is this for?

Reply:
Just a double check in case the behavior changes in aiohttp in the future